From 0f035466f4844fa048dd9e099cc97ec2d51b3516 Mon Sep 17 00:00:00 2001
From: Bai Ping <b51503@freescale.com>
Date: Fri, 28 Aug 2015 01:17:44 +0800
Subject: [PATCH 0419/5242] MLK-11437-01 ARM: dts: imx: correct the clock info
 of cpu node

commit  be775856209ddc3b5b0ab514553c42d8bb180ffc from
https://source.codeaurora.org/external/imx/linux-imx.git

The pll1_bypass and pll1_bypass_src clock index  in cpu node
should be IMX6SX_PLL1_BYPASS and IMX6SX_PLL1_BYPASS_SRC,
so correct this.

Signed-off-by: Bai Ping <b51503@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx.dtsi |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index 62b4595..82b3e9a 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -85,8 +85,8 @@
 				 <&clks IMX6SX_CLK_PLL1_SW>,
 				 <&clks IMX6SX_CLK_PLL1_SYS>,
 				 <&clks IMX6SX_CLK_PLL1>,
-				 <&clks IMX6SX_PLL7_BYPASS>,
-				 <&clks IMX6SX_PLL7_BYPASS_SRC>;
+				 <&clks IMX6SX_PLL1_BYPASS>,
+				 <&clks IMX6SX_PLL1_BYPASS_SRC>;
 			clock-names = "arm", "pll2_pfd2_396m", "step",
 				      "pll1_sw", "pll1_sys", "pll1",
 				      "pll1_bypass", "pll1_bypass_src";
-- 
1.7.9.5

